code/modules/mining/equipment/mining_tools.dm

```diff
@@ -121,6 +121,7 @@
 	hitsound = 'sound/weapons/sonic_jackhammer.ogg'
 	usesound = 'sound/weapons/sonic_jackhammer.ogg'
 	toolspeed = 0.1 //the epitome of powertools. extremely fast mining, laughs at puny walls
+	force = 20 //This thing breaks rwalls, it should be able to hit harder than a DIY bone pickaxe.
```

Comment:
This change is on the PR description?

Reply:
It is not. I'll include it as I update the PR description here to describe the items.